### PR TITLE
Angular mock server

### LINKED ISSUE #65: Setup Json Server

### PR DESCRIPTION
**Pull request by dudaskobendeguz**
closes #65

### CODE REVIEW

**Comment by dudaskobendeguz** [comment]
closes #65

File: angular/package.json [hunk 1]
```diff
@@ -18,6 +18,7 @@
     "@angular/platform-browser": "^14.1.0",
     "@angular/platform-browser-dynamic": "^14.1.0",
     "@angular/router": "^14.1.0",
+    "angular-in-memory-web-api": "^0.14.0",
```
Comment (benedekhalaj) [inline angular/package.json:21]: great addition! :)

File: angular/src/app/app.module.ts [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
 @NgModule({
   declarations: [
     AppComponent
   ],
   imports: [
     BrowserModule,
-    AppRoutingModule
+    AppRoutingModule,
+    HttpClientModule,
+    // The HttpClientInMemoryWebApiModule module intercepts HTTP requests
```
Comment (benedekhalaj) [inline angular/src/app/app.module.ts:19]: Thanks for commenting this

File: angular/src/app/services/in-memory-data.service.spec.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { TestBed } from '@angular/core/testing';
+
```
Comment (benedekhalaj) [inline angular/src/app/services/in-memory-data.service.spec.ts:2]: Nice generated test! :)

**Review by benedekhalaj — APPROVED** [review]
Great! :)

**Pull request merged**